src/components/uri/Visualization.tsx

```diff
@@ -391,7 +393,17 @@ export class VisualizationWrapped extends React.Component<
         this.visualizationUri = visualizationUri;
         this.exportTitle = get(mdObject, "meta.title", "");
 
-        const { afm, resultSpec } = buildAfmResultSpec(mdObject.content, visualizationType, locale);
+        const config: IChartConfig = this.createVisualizationConfig();
+        const canSortStackTotalValue: boolean =
+            get(config, "stackMeasures", false) &&
+            areAllMeasuresOnSingleAxis(mdObject.content, get(config, "secondary_yaxis", undefined));
+
+        const { afm, resultSpec } = buildAfmResultSpec(
+            mdObject.content,
+            visualizationType,
+            locale,
+            canSortStackTotalValue,
+        );
```

Comment:
it affect chart with interface
```
unsort expected
<BarChart
        projectId={projectId}
        measures={[totalSales]}
        viewBy={month}
    />
```
And
custom sort expected
```
<BarChart
        projectId={projectId}
        measures={[totalSales]}
        viewBy={month}
        sortBy={sortBy}
    />
```
But it always be modified as default sort of bar chart.
We should only apply for interface
```
<Visualization
        projectId="<project-id>"
        identifier="<visualization-identifier>"
        config={<chart-config>}
        sdk={<sdk>}
    />
```

Reply:
BarChart and URI Visualization are 2 different flow.
They are not affecting each other